templates/zerver/help/setting-up-zulip-for-a-class.md

```diff
@@ -145,12 +135,8 @@ A few settings to highlight:
 
 * [Add custom emoji](/help/custom-emoji) that your class will enjoy.
 
-
-[topic-editing-permissions]: /help/configure-message-editing-and-deletion
 [default-code-block-language]: /help/code-blocks#default-code-block-language
 [code-playgrounds]: /help/code-blocks#code-playgrounds
-[email-address-visibility]: /help/restrict-visibility-of-email-addresses
-[who-can-invite]: /help/restrict-account-creation#change-who-can-send-invitations
```

Comment:
These links were not being used in the article, so I removed them. Were not deleted when the content was copied from `include/customize-organization-settings.md` and then updated for this particular article content / needs.

Reply:
Makes sense!